var/spack/repos/builtin/packages/py-simwriter/package.py

```diff
@@ -12,7 +12,7 @@ class PySimwriter(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/project/proj1/simwriter'
     git      = 'ssh://bbpcode.epfl.ch/project/proj1/simwriter'
 
-    version('1.0.0', commit='eea126f3c0511342b9af78e51c7133aae1d28896')
+    version('1.0.0', commit='b2b560e80a369000aa0a1a2f70a52672a35ff5ed')
```

Comment:
[sic]… why?

Reply:
It is a moving target currently... Probably it will be a good candidate to have setuptools_scm but this will happen later.

Reply:
if that is moving target, why can't we bump version? (with the reproducibility and related, discussion we are having, adding new code with same version not a good idea)